content/en/tracing/llm_observability/tracing_concepts.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Tracing Concepts
+---
+
+<div class="alert alert-info">LLM Observability is in public beta.</a></div>
+
+## Spans
+
+A span is a unit of work representing an operation in your LLM application, and is the building block of a trace.
+
+### Span Attributes
+
+A span consists of the following attributes:
```

Comment:
What would you think about something easier to scan like:


- Name
- Start and end time 
- Error type, message, and traceback
- Inputs and outputs (e.g. LLM prompts and completions)
- Metadata (e.g. LLM parameters such as `temperature`, `max_tokens`)
- Metrics (e.g. `prompt_tokens`, `completion_tokens`)
- Tags